ElectronClient/gui/KeymapConfig/styles/index.ts

```diff
@@ -0,0 +1,51 @@
+import { KeymapConfigScreenProps } from '../KeymapConfigScreen';
+
+const { buildStyle } = require('lib/theme');
+
+export default function styles(props: KeymapConfigScreenProps) {
+	return buildStyle('KeymapConfigScreen', props.theme, (theme: any) => {
+		return {
+			containerStyle: {
+				...theme.containerStyle,
+				padding: 15,
+			},
+			topActionsStyle: {
+				display: 'flex',
+				flexDirection: 'row',
+			},
+			inlineButtonStyle: {
+				...theme.buttonStyle,
+				padding: 0,
+				marginLeft: 12,
+			},
+			filterInputStyle: {
```

Comment:
It's a minor thing but please don't name the style elements as "somethingStyle" since we know it's a style. So "containerStyle" => "container", "topActionsStyle" => "topActions", etc.